invenio_oaiserver/models.py

```diff
@@ -76,5 +80,16 @@ class OAISet(db.Model, Timestamp):
     )
     """Search pattern to get records."""
 
+    @hybrid_property
```

Comment:
simple validator would do it - see http://docs.sqlalchemy.org/en/latest/orm/mapped_attributes.html#simple-validators